doc/lsp.txt

```diff
@@ -498,6 +498,10 @@ usePopupInCodeAction    |Boolean| option.  When using the |:LspCodeAction|
 			command to display the code action for the current
 			line, use a popup menu instead of echoing.
 			By default this is set to false.
+						*lsp-opt-useBufferCompletion*
+useBufferCompletion     |Boolean| option. If enable adds current buffer words
+			to autocompletion list. Мay degrade vim performance.
+			By default this is set to false.
```

Comment:
Can you change the help text to:

If enabled, the words from the current buffer are added to the auto completion list.  This may degrade Vim performance as the current buffer contents are processed every time the completion menu is displayed.

Reply:
added fix in https://github.com/yegappan/lsp/pull/284/commits/b48c94b90e9da99d6fa271f0eba8400cb5d11d48